R/data_merge_module.R

```diff
@@ -86,7 +86,7 @@
 #'       datasets = datasets,
 #'       merge_function = "dplyr::left_join"
 #'     )
-#'     output$expr <- renderText(merged_data()$expr)
+#'     output$expr <- renderText(paste(merged_data()$expr, collapse = "\n"))
```

Comment:
In data_merge_srv there is `merged_data$expr <- paste(merged_data$expr, collapse = "\n")`. SO paste is not needed


```suggestion
#'     output$expr <- renderText(merged_data()$expr)
```